src/backend/gog/setup.ts

```diff
@@ -123,7 +130,31 @@ async function setup(
             keyCommand = `/d "${valueData}" /v "${valueName}" /t ${regType}`
           }
           // Now create a key
-          const command = `${commandPrefix} reg add "${registryPath}" ${keyCommand} /f /reg:32`
+          let command = `${commandPrefix} reg add "${registryPath}" ${keyCommand} /f /reg:32`
+          if (gameSettings.wineVersion.type === 'bottles') {
+            if (!valueName && !valueData) {
+              break // Skip creating just registry path, bottles CLI needs some key to add
```

Comment:
Every registry folder has the `(Default)` key (type `REG_SZ` with no data set), can't you add that?

Reply:
Empty registry keys are not needed either way. There are always other instructions that set some value in the key